src/git/pack_engine.ml

```diff
@@ -776,8 +776,8 @@ module Make
          | Ok ()          -> sys_err
          | Error sys_err' ->
            Log.err (fun l ->
-               l "Impossible to close the fd of the PACK file (to analyze it): \
-                  %a." FS.pp_error sys_err');
+               l "Impossible to close the fd of the PACK file (to analyze it), \
+                  ignoring: %a." FS.pp_error sys_err');
```

Comment:
Do we really want to display that here, or should that message simply be in `pp_error` and we let the user print the error if needed? I think it's better to not print anything as long as the error is not discarded and let the use decide later to print/ignore the error.